renovate.json

```diff
@@ -1,11 +1,22 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": [
-    "config:base"
-  ],
+  "extends": ["config:base"],
   "platformAutomerge": true,
   "pre-commit": {
     "enabled": true
   },
-  "enabledManagers": ["pre-commit"]
+  "enabledManagers": ["pre-commit", "terraform"],
+  "terraform": {
+    "fileMatch": ["^/terraform/.*\\.tf$"]
+  },
+  "packageRules": [
```

Comment:
Is there a reason these two things specifically have their own rules?

Reply:
So alexbasista/workspacer/tfe module we specify in basically in each file in the "tac-configuration" project - we don't want to update all of those individually.

Probably doesn't make sense to group the hashicorp/tfe - removed.